select_assignment2_flicks/Model/Movie.swift

```diff
@@ -8,17 +8,27 @@
 
 import UIKit
 import SwiftyJSON
+import RealmSwift
+
+class MovieRealm : Object {
+    var id: String = ""
+}
 
 struct Movie {
     
     init(json: JSON) {
+        self.id = json["id"].stringValue
         self.title = json["title"].stringValue
         self.imageUrl = json["poster_path"].stringValue
         self.voteAverage = json["vote_average"].doubleValue
         self.overview = json["overview"].stringValue
         self.releaseDate = json["release_date"].stringValue
     }
     
+    let realm = try! Realm()
```

Comment:
a tiny bit messy and wasteful to have each model have their own reference to realm but not a huge deal. maybe a static instance?